drivers/flash/flash_stm32.c

```diff
@@ -37,8 +37,8 @@ LOG_MODULE_REGISTER(flash_stm32, CONFIG_FLASH_LOG_LEVEL);
 
 static const struct flash_parameters flash_stm32_parameters = {
 	.write_block_size = FLASH_STM32_WRITE_BLOCK_SIZE,
-	/* Some SoCs (L0/L1) use an EEPROM under the hood. Distinguish
-	 * between them based on the presence of the PECR register. */
+/* Some SoCs (L0/L1) use an EEPROM under the hood. Distinguish
+ * between them based on the presence of the PECR register. */
```

Comment:
Please discard this change. Indentation was fine (even if coding style was not :)